plugins/modules/purefa_pgsnap.py

```diff
@@ -917,7 +924,7 @@ def main():
     )
 
     required_if = [("state", "copy", ["suffix", "restore"])]
-    mutually_exclusive = [["now", "remote"]]
+    mutually_exclusive = [["now", "remote"], ["overwrite", "add_to_pgs"]]
```

Comment:
@sdodsley , Shouldn't we consider adding "with_default_protection" as well.
While am testing in interpreter, passing it also resulted in invalid param combo.

Reply:
Agreed and done